README.md

```diff
@@ -27,7 +27,9 @@ This repo contains examples of PyPRT usage. You can find installation instructio
 
 PyPRT can be installed with `pip install pyprt` or `conda install -c esri pyprt`. To run the examples a few more packages are required, so please read on! :-)
 
-(*) A note regarding Python 3.9 and Conda: The arcgis package for Python 3.9 is not yet available in Conda, therefore the examples 8 and 9 are not working with Python 3.9 in a Conda environment.
+(*) Notes regarding Python 3.9 and Conda:
```

Comment:
Consider removing the `(*)`, we do not link to a text location above in this case.